src/cryogen_core/util.clj

```diff
@@ -60,3 +62,24 @@
   "Converts a java.io.File to a java.net.URL."
   [^java.io.File f]
   (.. f (getAbsoluteFile) (toURI) (toURL)))
+
+(defn parse-post-date
+  "Parses the post date from the post's file name and returns the corresponding java date object"
+  ;; moved from cryogen-core.compile
+  [^String file-name date-fmt]
+  (let [fmt (java.text.SimpleDateFormat. date-fmt)
+        c (count date-fmt)]
+    (try (if-let [last-slash (last-index-of file-name "/")]
+      (.parse fmt (.substring file-name (inc last-slash) (+ last-slash (inc c))))
+      (.parse fmt (.substring file-name 0 c)))
+         (catch Exception e
+           (throw (ex-info "Failed to parse date from filename"
+                          {:date-fmt date-fmt
+                           :file-name file-name}
+                          e)))))
+)
```

Comment:
minor formatting nitpick, can we move the paren to the previous line? :)

Reply:
Will do.